doc/xml/user-guide.xml

```diff
@@ -558,7 +558,13 @@
 
     <!-- ======================================================================================================================= -->
     <block-define if="'{[package]}' eq 'none'" id="br-install">
-        <p><backrest/> needs to be installed from a package or installed manually as shown here.</p>
+        <p>Installing <backrest/> from a package is preferable to building from source. When installing from a package the rest of the instructions in this section are generally not required, but it is possible that a package will skip creating one of the directories or apply incorrect permissions. In that case it may be necessary to manually create directories or update permissions.</p>
+
+        <p if="{[os-type-is-debian]}">{[user-guide-os]} packages for <backrest/> are available at <link url="https://www.postgresql.org/download/linux/ubuntu/">apt.postgresql.org</link>.</p>
+
+        <p if="{[os-type-is-rhel]}">{[user-guide-os]} packages for <backrest/> are available from <link url="{[crunchy-url-base]}">Crunchy Data</link> or <link url="http://yum.postgresql.org">yum.postgresql.org</link>.</p>
+
+        <p>If packages are not provided for your distribution/version then you can <link section="/build">build from source</link> and then install manually as shown here.</p>
```

Comment:
maybe avoid the second "then" in the sentence?

Reply:
I decided to remove the first then instead in 28dd1abc4ca. Look OK?

Reply:
Looks great, thanks!